LocationServices/LocationServices/Coordinators/ChildCoordinators/SplitView/SplitViewGeofencingMapCoordinator.swift

```diff
@@ -0,0 +1,171 @@
+        floatingView?.setupNavigationSearch(state: state)
+    }
+}
+
+extension SplitViewGeofencingMapCoordinator: GeofenceNavigationDelegate {
+    func dismissCurrentScene(geofences: [GeofenceDataModel], shouldDashboardShow: Bool) {
+        if shouldDashboardShow {
+            showDashboardFlow(geofences: geofences, lat: nil, long: nil)
+        } else {
+            splitDelegate?.showOnlySecondary()
+            supplementaryNavigationController?.popToRootViewController(animated: false)
+        }
+    }
+    
+    func showDashboardFlow(geofences: [GeofenceDataModel], lat: Double?, long: Double?) {
+        supplementaryController.userlocation = (lat, long)
+        supplementaryController.datas = geofences
+        supplementaryController.viewModel.geofences = geofences
+        
+        supplementaryNavigationController?.popToRootViewController(animated: false)
+        splitDelegate?.showSupplementary()
+    }
+    
+    func showAddGeofenceFlow(activeGeofencesLists: [GeofenceDataModel],
+                             isEditingSceneEnabled: Bool = false,
+                             model: GeofenceDataModel?,
+                             lat: Double?,
+                             long: Double?) {
+        let controller = AddGeofenceBuilder.create(activeGeofencesLists: activeGeofencesLists,
+                                                   isEditingSceneEnabled: isEditingSceneEnabled,
+                                                   model: model,
+                                                   lat: lat,
+                                                   long: long)
+        controller.delegate = self
+        
+        supplementaryNavigationController?.pushViewController(controller, animated: true)
+        splitDelegate?.showSupplementary()
+    }
+    
+    func showMapStyleScene() {
+        let controller = ExploreMapStyleBuilder.create()
+        
+        controller.modalPresentationStyle = .formSheet
+        controller.isModalInPresentation = true
+        controller.dismissHandler = { [weak self] in
+            self?.splitViewController.dismiss(animated: true)
+        }
+        if let sheet = controller.sheetPresentationController {
+            sheet.preferredCornerRadius = 10
+        }
+        
+        splitViewController.present(controller, animated: true)
+    }
+    
+    func showLoginFlow() {
+        (UIApplication.shared.delegate as? AppDelegate)?.navigationController = supplementaryNavigationController
+        
+        let controller = LoginVCBuilder.create()
+        controller.dismissHandler = { [weak self] in
+            self?.splitViewController.dismiss(animated: true)
+        }
+        
+        controller.postLoginHandler = { [weak self] in
+            self?.showLoginSuccess()
+        }
+        
+        controller.modalPresentationStyle = .formSheet
+
+        if let sheet = controller.sheetPresentationController {
+            sheet.preferredCornerRadius = 10
```

Comment:
All preferred corner radius move to const.

Reply:
fixed